ivy/functional/ivy/losses.py

```diff
@@ -266,3 +266,45 @@ def sparse_cross_entropy(
     """
     true = ivy.one_hot(true, pred.shape[axis])
     return ivy.cross_entropy(true, pred, axis=axis, epsilon=epsilon, out=out)
+
+sparse_cross_entropy.unsupported_dtypes = {"torch": ("float16",)}
+sparse_cross_entropy.supported_dtypes = {"tensorflow": ("uint8", "int32", "int64")}
+
+
+@handle_nestable
```

Comment:
Instead of adding a new method as a part of the functional API, you should create a composition in the frontend itself as explained [here.](https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends.html#temporary-compositions) and [here](https://lets-unify.ai/ivy/contributing/4_open_tasks.html#frontend-apis)

Reply:
Like my reponse above, code related to cross_entropy isn't writen by me.